tiatoolbox/dataloader/wsireader.py

```diff
@@ -53,7 +53,8 @@ class WSIReader:
     """
 
     def __init__(
-        self, input_img=".",
+        self,
+        input_img=".",
```

Comment:
Did we agree on removing default input for input_img?

Reply:
Updated in this commit https://github.com/TIA-Lab/tiatoolbox/commit/5276627cb0a863ab58b0e63001e9ad1b8dac5c3a